### PR TITLE
update DL skills

### PR DESCRIPTION
**Pull request by douglowe**

### CODE REVIEW

File: people.json [hunk 1]
```diff
@@ -316,15 +316,20 @@
...
             "High-performance computing",
             "Fortran",
             "Bash (Unix shell)",
-            "National Center for Atmospheric Research",
             "Python (programming language)",
             "Perl",
             "MATLAB",
             "Make (software)",
+            "Docker (software)",
+            "Singularity (software)"
```
Comment (rowleya) [inline people.json:330]:
```suggestion
            "Singularity (software)",
```

**Review by rowleya — APPROVED** [review]

**Comment by rowleya** [comment]
Looks fine to me now (minor correction) but I think Ian likes to do the final merge as it triggers deployment...

**Comment by ianhinder** [comment]
Nice to see the automated check on JSON validity being useful!

**Comment by ianhinder** [comment]
Yes; the final merge into master triggers the deployment, so ideally I like to do this myself so that I can watch it happening and debug it if it fails.  Probably I should add a branch protection rule.

**Pull request merged**